extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -1,37 +1,879 @@
+
+### Tile Availability
+
+**Tile availability** is a bitstream that determines which tiles exist within a subtree. There is one bit for each subtree node. A 1 indicates that a tile is available, while a 0 indicates that a tile is unavailable. The bits follow [Morton Order](#morton-order).
+
+![Tile Availability](figures/tile-availability.jpg)
+
+In the diagram above, colored nodes indicate available tiles, while nodes with dashed outlines are unavailable tiles Note that this is for illustration purposes only. Binary trees are not supported.
+
+If a tile is marked as available, it may have extensions attached to it as defined by the tileset author, and it may have content or children.
+
+If a non-root tile's availability is 1, its parent tile's availability must also be 1. `tileAvailability.constant: 0` is disallowed, as subtrees must have at least one available tile.
+
+### Content Availability
+
+**Content availability** is a bitstream, separate from tile availability, that determines which tiles have an associated content 3D model. Like tile availability, there is one bit for each tile. A 1 indicates a content file exists for this tile, while a 0 indicates that no content file exists. An available tile does not need to have content if, for example, the tileset author defines extensions that store extra information in tiles outside of content.
+
+![Content Availability](figures/content-availability.jpg)
+
+The purpose of content availability is to check if a content 3D model exists before making a network request. If content is marked as unavailable, the network request for that file must be skipped.
+
+If content availability is 1 its corresponding tile availability must also be 1. Otherwise, it would be possible to specify content files that are not reachable by the tiles of the tileset. 
+
+If content availability is 0 and its corresponding tile availability is 1 then the tile is considered to be an empty tile.
+
+When a subtree has at least one tile with content, content availability is required. If no tile in the subtree has content, then content availability is disallowed.
+
+### Child Subtree Availability
+
+**Child subtree availability** is a bitstream that determines which subtrees can be reached from the deepest level of this subtree.
+
+The child subtree availability bitstream has slightly different structure than tile or content availability. There is one bit for each node in the level of the tree immediately below the subtree. That is, if the deepest subtree is level `L`, then there is one bit for every node at level `L + 1`. Since each node at level `L` has `N` children (4 for `QUADTREE` or 8 for `OCTREE`), there are `N * (L + 1)` bits in the child subtree availability bitstream. A 1 bit means there is a child subtree available at that position in the tree. A 0 bit means there is no subtree available at that position.
```

Comment:
* `That is, if the deepest subtree`. Should it be `deepest subtree node`?
* I assume `L` is a level local to the subtree, but it could be read as a global level.
* The equation `N * (L + 1)` should be `N ^ (L + 1)`.
* No mention of morton encoding
 
I feel like this paragraph would be more clear if defined in terms of `subtreeLevels`